src/components/backupWallet/ExportWalletData.js

```diff
@@ -55,16 +50,6 @@ const ExportWalletData = ({ navigation, styles, theme }: ExportWalletProps) => {
     ]
   }, [goodWallet])
 
-  const onPublicKeyCopied = useCallback(
-    () =>
-      showDialog({
-        showButtons: false,
-        onDismiss: noop,
```

Comment:
why just not set onDismiss: hideDialog here ? and that's all the fix!

Reply:
Because we also need to call _onCopied from BorderedBox which related to various local states from BorderedBox (the previous implementation of the popup call is at the outer level). Also it adds ability to ask confirmation for secure data with just one prop